### ISSUE TITLE
Warmup 3's "Search Through a File" exercise is confusing

### ISSUE THREAD

**Issue by yeeunmariakim** [gkapfham/www.warmups.dev#15]
At the end of the exercise, it assigns the following tasks:

```
# TODO: search for a missing email address
matching_emails = search(emails_text, "___________________")
display(matching_emails)
# TODO: search for an available email address
matching_emails = search(emails_text, "__________________")
```

This is confusing because there is no way to know from the code what is a "missing" or "available" email. The only way to know this is to remember that the program uses the same file as the previous exercise and go back to the previous exercise and complete it to get the list of emails.

Moreover, it looks like the solution does not care about whether the emails are actually missing or available. For instance, this is the output from a "correct" solution.

```
🔎 Searching for example1@gmail.com
😢 Sorry, no matches!
🔎 Searching for example2@gmail.com
😢 Sorry, no matches!
✔ Well done!
```

Actually, just removing the TODO markers will produce a correct solution.

**Comment by DayMath82** [comment]
I agree, I guess that would have to do with the saving of the warm-ups, and it gets repetitive to go back to the previous modules to find the emails. But either way it's a matter of making sure your code is right. The reason why removing the TODO markers would make the right answer is because the code that would make it correct has nothing to do with the specific emails, rather if it's right. I think all that needs to be done is to have 2 set emails in the directions for the user to use in their code and what they should have as their output.
